puma_motor_driver/src/driver.cpp

```diff
@@ -196,7 +203,7 @@ void Driver::verifyParams()
       if (fabs(getP() - gain_p_) < 0.00001)
       {
         state_++;
-        ROS_INFO("Dev: %i P gain constant was set to %f.", device_number_, gain_p_);
+        ROS_INFO("Dev: %i P gain constant was set to %f.", device_number_, getP());
```

Comment:
Is it necessary to use the accessor here, since you're in the class?

Also, let's make all these outputs `ROS_DEBUG`— the info stream should really be limited to one or two messages on a successful initialization.